src/basis/basis.js

```diff
@@ -378,6 +379,23 @@ export class Basis {
             .some(x => !x);
     }
 
+    /**
+     * @summary function returns the minimum basis lattice size for a structure.
+     * The lattice size is based on the atomic radius of an element if the basis contains a single atom.
+     * The lattice size is based on the maximum pairwise distance across a structure if the basis contains > 2 atoms.
+     * @returns {Number}
+     */
+    getMinimumBasisLatticeSize() {
+        let latticeSizeAdditiveContribution = 0;
+        if (this._elements.array.length === 1) {
+            const elementSymbol = this._elements.getArrayElementByIndex(0)
+            latticeSizeAdditiveContribution = getElementAtomicRadius(elementSymbol);
```

Comment:
`getElementAtomicRadius` function was moved to Periodic-Table - https://github.com/Exabyte-io/periodic-table.js/pull/11/files